### ISSUE TITLE
hang during "Generating output files..." phase

### ISSUE THREAD

**Issue by derrickoswald** [jsdoc3/jsdoc#1048]
For the group of files in the linked zip https://www.dropbox.com/s/606cqgw9fzhwkow/jsdoc_bug.zip?dl=0, jsdoc (JSDoc 3.3.2 (Sat, 13 Jun 2015 22:20:28 GMT)) hangs at the point where it is generating output files. Removing any one of the files makes the problem go away.

```bash
$ /usr/local/bin/jsdoc --verbose js/ 
Parsing xxxx/js/about.js ...complete.
Parsing xxxx/js/chooser.js ...complete.
Parsing xxxx/js/configuration.js ...complete.
Parsing xxxx/js/database.js ...complete.
Parsing xxxx/js/deluge.js ...complete.
Parsing xxxx/js/discover.js ...complete.
Parsing xxxx/js/keybase.js ...complete.
Parsing xxxx/js/torrent.js ...complete.
Parsing xxxx/js/wizard.js ...complete.
Generating output files...
```

**Comment by hegemonic** [comment]
Confirmed. GitHub master does not have this issue.

**Comment by hegemonic** [comment]
Looks like this was fixed by 99fb8a00d2fa986ee0a62347863fdc1e53e99ffc. I've cherry-picked that commit to the `releases/3.3` branch, so JSDoc 3.3.3 will include the fix for this bug. For now, you can use the `releases/3.3` branch to get the fix.

Thanks for the heads up!
